modules/hello-world/examples/index_hello_world.py

```diff
@@ -0,0 +1,87 @@
+from datetime import timedelta
+
+from couchbase.auth import PasswordAuthenticator
+from couchbase.cluster import Cluster, ClusterOptions
+from couchbase.management.queries import (CreatePrimaryQueryIndexOptions,
+                                          CreateQueryIndexOptions,
+                                          DropPrimaryQueryIndexOptions,
+                                          WatchQueryIndexOptions)
+
+cluster = Cluster('couchbase://localhost', ClusterOptions(
+    PasswordAuthenticator('Administrator', 'password'))
+)
+
+print("[primary]")
+# tag::primary[]
+cluster.query_indexes().create_primary_index(
+    "travel-sample",
+    # Don't error if the primary index already exists.
+    CreatePrimaryQueryIndexOptions(ignore_if_exists=True)
+)
+# end::primary[]
+print("Index creation complete")
+
+print("\n[secondary]")
+# tag::secondary[]
+cluster.query_indexes().create_index("travel-sample", "index_name", ["name"])
+# end::secondary[]
+print("Index creation complete")
+
+print("\n[composite]")
+# tag::composite[]
+cluster.query_indexes().create_index(
+    "travel-sample",
+    "index_travel_info",
+    ["name", "id", "icao", "iata"]
+)
+# end::composite[]
+print("Index creation complete")
+
+print("\n[drop-primary]")
+# tag::drop-primary[]
+cluster.query_indexes().drop_primary_index(
+    "travel-sample",
+    DropPrimaryQueryIndexOptions(index_name="#primary")
+)
+# end::drop-primary[]
+print("Primary index deleted successfully")
+
+print("\n[drop-secondary]")
+# tag::drop-secondary[]
+cluster.query_indexes().drop_index("travel-sample", "index_name")
+# end::drop-secondary[]
+print("Index deleted successfully")
+
+print("\n[defer-create]")
+# tag::defer-create-primary[]
+cluster.query_indexes().create_primary_index(
+    "travel-sample",
+    CreatePrimaryQueryIndexOptions(index_name="#primary", deferred=True)
+)
+# end::defer-create-primary[]
+
+# tag::defer-create-secondary[]
+cluster.query_indexes().create_index(
+    "travel-sample",
+    "idx_name_email",
+    ["name", "email"],
+    CreatePrimaryQueryIndexOptions(deferred=True)
```

Comment:
Creating a secondary index. I think this should be:

```suggestion
    CreateQueryIndexOptions(deferred=True)
```